tsconfig.json

```diff
@@ -28,5 +28,5 @@
         "lib": ["dom", "es2019"]
     },
     "include": ["frontend/**/*"],
-    "exclude": ["node_modules/**/*", "staticfiles/**/*", "frontend/dist/**/*"]
+    "exclude": ["node_modules/**/*", "staticfiles/**/*", "frontend/dist/**/*", "plugin-server/**/*"]
```

Comment:
I imagine we will want to turn on tsconfig for `plugin-server` or maybe the `tsconfig.json` in that directory will take over for that directory?